python/samples/concepts/local_models/lm_studio_chat_completion.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+
+import asyncio
+
+from openai import AsyncOpenAI
+
+from semantic_kernel.connectors.ai.open_ai.services.open_ai_chat_completion import OpenAIChatCompletion
+from semantic_kernel.contents.chat_history import ChatHistory
+from semantic_kernel.functions.kernel_arguments import KernelArguments
+from semantic_kernel.kernel import Kernel
+
+# This concept sample shows how to use the OpenAI connector to create a
+# chat experience with a local model running in LM studio: https://lmstudio.ai/
+# Please follow the instructions here: https://lmstudio.ai/docs/local-server to set up LM studio.
+# The default model used in this sample is phi3 due to its compact size.
+
+system_message = """
+You are a chat bot. Your name is Mosscap and
+you have one goal: figure out what people need.
+Your full name, should you need to know it, is
+Splendid Speckled Mosscap. You communicate
+effectively, but you tend to answer with long
+flowery prose.
+"""
+
+kernel = Kernel()
+
+service_id = "local-gpt"
+
+openAIClient: AsyncOpenAI = AsyncOpenAI(
+    api_key="fake-key",  # This cannot be an empty string, use a fake key
+    base_url="http://localhost:1234/v1",
+)
+kernel.add_service(OpenAIChatCompletion(service_id=service_id, ai_model_id="phi3", async_client=openAIClient))
```

Comment:
what would it take to just pass the same info into the OpenAIChatCompletion directly?

Reply:
This is needed since the local models require different endpoints than the default OpenAI one.